docs/development.rst

```diff
@@ -1,6 +1,91 @@
 .. _development:
 
+1. System requirement :
 
+Any system with basic configuration.
+Operating System : Any (Windows / Linux / Mac).
+
+2. Software requirement :
+Updated browser
+python
+
+3. Skill set :
+Knowledge of git & github.
+python
+
+============================
```

Comment:
The other ReST files use only lines below the heading. Please keep it consistent.

Reply:
this is a kind of "chapter" style, or? But I also think we should not overuse this. And may be later do a refactoring where it is better used.